src/main/java/com/linecorp/armeria/client/http/ImmutableHttpHeaders.java

```diff
@@ -37,7 +37,7 @@
 
     @Override
     public String get(String name) {
-        return delegate.get(name);
+        return delegate.getAsString(name);
```

Comment:
I find it delegating as is better because `ImmutableHttpHeaders` is just a wrapper class that is intended to delegate read-only operations of `HttpHeaders`. Please also note that `ImmutableHttpHeaders` is package-private and thus there's no chance of users' seeing it ever. No matter what you do, a user will see `HttpHeaders` and it's up to a user to avoid calling a deprecated method.